app/views/providers/check_client_details/show.html.erb

```diff
@@ -1,41 +1,65 @@
-<%= page_template page_title: t(".title"), template: :basic do %>
-  <%= form_with(
-        model: @form,
-        url: providers_legal_aid_application_check_client_details_path(@legal_aid_application),
-        method: :patch,
-        local: true,
-      ) do |form| %>
+<% page_title = @legal_aid_application.applicant.has_partner? ? t(".partner_title") : t(".client_only_title")
+   checking_details_for_dwp = true %>
+<%= page_template page_title:, template: :basic do %>
 
     <%= govuk_fieldset_header page_title %>
 
-    <table class="govuk-table">
-      <caption class="govuk-table__caption govuk-table__caption--m govuk-visually-hidden"><%= t(".table_caption") %></caption>
-      <thead class="govuk-table__head">
-      </thead>
-      <tbody class="govuk-table__body">
-      <tr class="govuk-table__row">
-        <th scope="row" class="govuk-table__header"><%= t(".name") %></th>
-        <td class="govuk-table__cell"><%= "#{@applicant.first_name} #{@applicant.last_name}" %></td>
-      </tr>
-      <tr class="govuk-table__row">
-        <th scope="row" class="govuk-table__header"><%= t(".date_of_birth") %></th>
-        <td class="govuk-table__cell"><%= @applicant.date_of_birth.strftime("%e %B %Y") %></td>
-      </tr>
-      <tr class="govuk-table__row">
-        <th scope="row" class="govuk-table__header"><%= t(".national_insurance_number") %></th>
-        <td class="govuk-table__cell"><%= @applicant.national_insurance_number.gsub(/(.{2})(?=.)/, '\1 \2') %></td>
-      </tr>
-      </tbody>
-    </table>
-
-    <% options = [[true, t(".details_correct")], [false, t(".details_incorrect")]] %>
-
-    <%= form.govuk_collection_radio_buttons :check_client_details,
-                                            options,
-                                            :first,
-                                            :second,
-                                            legend: nil %>
-
-    <%= next_action_buttons(show_draft: true, form:) %>
-  <% end %>
+    <br>
+
+    <h2 class="govuk-heading-m"><%= t(".h2_client") %></h2>
```

Comment:
nit: It may be ideal to use [govuk spacing](https://design-system.service.gov.uk/styles/spacing/#static-spacing) overrides on the `<h2>` instead of a `<br>`